packages/functional-tests/tests/settings/totp.spec.ts

```diff
@@ -1,12 +1,12 @@
 import { test, expect } from '../../lib/fixtures/standard';
 import { EmailHeader, EmailType } from '../../lib/email';
 
-test.describe('severity-1 #smoke', () => {
+test.describe('two step auth', () => {
   // https://testrail.stage.mozaws.net/index.php?/cases/view/1293446
   // https://testrail.stage.mozaws.net/index.php?/cases/view/1293452
-  test('add and remove totp #1293446 #1293452', async ({
+  test('add and remove totp', async ({
```

Comment:
Same here and L47 below. Do we want to retain the testrail annotations here?

Reply:
I removed it to be consistent with our structure. As with the new tests that are being written we don't have reference to the testrails cases anywhere.